GDSCommon-Demo/GDSCommon-Demo/cy.lproj/LaunchScreen.strings

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this empty file here intentionally?

Reply:
To get the actual welsh string to show up in the Demo, I had to add the localisation (below). If I didnt select any files it didnt work :( so I think we need this here. 
<img width="538" alt="image" src="https://github.com/govuk-one-login/mobile-ios-common/assets/39125995/b2db0839-0399-430a-a781-283035423d74">